Game/Source/GDKTestGyms/Private/UserExperienceReporter.cpp

```diff
@@ -96,6 +97,10 @@ void UUserExperienceReporter::ReportMetrics()
 			if (Constants->ClientFPSMetricDelay.HasTimerGoneOff() && GameInstance->GetAveragedFPS() < Constants->GetMinClientFPS())
 			{
 				bFrameRateValid = false;
+				if (const USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(GameInstance->GetWorld()->GetNetDriver()))
```

Comment:
Probably want some log even if netdriver is null. Maybe the worker id can just be invalid if null driver.